test/runtests.sh

```diff
@@ -21,6 +21,8 @@ for i in test/00/*.sh; do
     # t0029a.sh tests that saving over a readonly file is prevented. Test doesn't work as root
     elif [ `whoami` = 'root' -a $i = test/00/t0029a.sh ]; then
         su minsky $i &> /dev/null
+    elif [ $i = test/00/findDefinition.sh ]; then
+        sh $i
```

Comment:
This is temporary - I;m trying to chase down a mysterious regression failure on Travis.